services/kaze/middlewares/constants.go

```diff
@@ -0,0 +1,8 @@
+package middlewares
+
+const (
+	// AppReqParam is Request param label for application instance type
+	AppReqParam = "app"
+	// DBReqParam Request param label for database instance type
+	DBReqParam = "db"
```

Comment:
These 2 variables are just used inside the `middlewares` package hence they need not be exported i.e they should begin with small letter

Reply:
done